src/test/java/uk/nhs/prm/e2etests/performance/RepoInPerformanceTest.java

```diff
@@ -183,16 +203,4 @@ private String getSmallMessageWithUniqueConversationIdAndMessageId(String messag
         message = message.replaceAll("__MESSAGE_ID__", messageId);
         return message;
     }
-
-    private AutoRefreshingRoleAssumingSqsClient appropriateAuthenticationSqsClient() {
-        if (config.performanceTestTimeout() > TestConfiguration.SECONDS_IN_AN_HOUR * 0.9) {
-            var authStrategyWarning = "Performance test timeout is approaching an hour, getting where this will not work if " +
-                    "using temporary credentials (such as obtained by user using MFA) if it exceeds the expiration time. " +
-                    "Longer runs will need to be done in pipeline where refresh can be made from the AWS instance's " +
-                    "metadata credentials lookup.";
-            System.err.println(authStrategyWarning);
-        }
-        out.println("AUTH STRATEGY: using auto-refresh, role-assuming sqs client");
-        return context.getBean(AutoRefreshingRoleAssumingSqsClient.class);
-    }
```

Comment:
Has this been replaced by something else? I wonder if this is necessary when running e2e tests on GoCD, as tests may take a long time to finish, which is maybe why they have this credential refreshing logic?

Reply:
TODO within refactoring ticket